src/main/resources/db/changelog/2023/10/10-01-changelog.yaml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1696145356872-3
+      author: joejoe2 (generated)
+      objectQuotingStrategy: QUOTE_ONLY_RESERVED_WORDS
+      changes:
+        - addColumn:
+            columns:
+              - column:
+                  defaultValueBoolean: false
+                  name: is_user1blocked
```

Comment:
is_user1blocked -> is_first_user_blocked

Reply:
Ok, i will adjust that